code/game/jobs/role_authority.dm

```diff
@@ -566,6 +566,9 @@ I hope it's easier to tell what the heck this proc is even doing, unlike previou
 			if(squad.put_marine_in_squad(human))
 				return
 
+		else if(squad.equivalent_name == preferred_squad) //equivalent names so that USCM names are used for squads of other factions
```

Comment:
this can just be in the above check ,if(squad.name == preferred_squad || squad.equivalent_name == preferred_squad)